dev-tools/packaging/packages.yml

```diff
@@ -449,6 +449,16 @@ shared:
       from: 'arm64v8/centos:7'
       buildFrom: 'arm64v8/centos:7'
 
+  - &cloud_agent_docker_spec
+    <<: *agent_docker_spec
+    extra_vars:
+      image_name: '{{.BeatName}}-cloud'
+
+  - &cloud_agent_docker_arm_spec
+    <<: *agent_docker_arm_spec
+    extra_vars:
+      image_name: '{{.BeatName}}-cloud'
```

Comment:
I think we can remove this block, an use the basic cloud_spec in the ARM definition, as we already do in https://github.com/elastic/beats/pull/27609/files#diff-9e8bcd1359f1e06bbdf2c89db711c6c400ed4142cf9764ff03e6e963013122e5R1055
```suggestion
```